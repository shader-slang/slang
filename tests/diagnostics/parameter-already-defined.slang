//DIAGNOSTIC_TEST:SIMPLE:
// re-use parameter name

int foo( int a, float a ) { return 0; }
