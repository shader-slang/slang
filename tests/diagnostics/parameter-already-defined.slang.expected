result code = -1
standard error = {
tests/diagnostics/parameter-already-defined.slang(4): error 30200: declaration of 'a' conflicts with existing declaration
int foo( int a, float a ) { return 0; }
                      ^
tests/diagnostics/parameter-already-defined.slang(4): note: see previous declaration of 'a'
}
standard output = {
}
